synapse/tests/test_lib_grammar.py

```diff
@@ -1269,10 +1269,10 @@ def test_cmdrargs(self):
         q = '''add {inet:fqdn | graph 2 --filter { -#nope } } inet:f-M +1 { [ graph:node='*' :type=m1]}'''
         correct = (
             'add',
-            '{inet:fqdn | graph 2 --filter { -#nope } }',
```

Comment:
is this no longer valid?

Reply:
Only in the case of argv values being pre-split. The leading/trailing curly braces are no longer correct in-band.